swarm/storage/feed/lookup/epoch_test.go

```diff
@@ -1,34 +1,40 @@
 package lookup_test
 
 import (
+	"fmt"
 	"testing"
 
+	"github.com/ethereum/go-ethereum/common/hexutil"
 	"github.com/ethereum/go-ethereum/swarm/storage/feed/lookup"
+	"github.com/ethereum/go-ethereum/swarm/testutil"
 )
 
-func TestMarshallers(t *testing.T) {
+func TestMarshallers(tx *testing.T) {
+	t := testutil.BeginTest(tx, false)
+	defer t.FinishTest()
 
 	for i := uint64(1); i < lookup.MaxTime; i *= 3 {
 		e := lookup.Epoch{
 			Time:  i,
 			Level: uint8(i % 20),
 		}
 		b, err := e.MarshalBinary()
-		if err != nil {
-			t.Fatal(err)
-		}
+		t.Ok(err)
+		t.EqualsKey(fmt.Sprintf("epoch%018d", i), hexutil.Bytes(b))
```

Comment:
@jpeletier it is not obvious to a new-comer what this is doing. I suggest renaming this to something related to `golden` files - I think this is the most common name for such kind of tests in Go.

One post documenting them is https://medium.com/soon-london/testing-with-golden-files-in-go-7fccc71c43d3

Maybe: `t.CompareKeyWithGoldenFile` ? Although that is a bit long... to me `t.EqualsKey` suggests that you are comparing the two parameters.